pytorch_lightning/core/lightning.py

```diff
@@ -112,6 +113,8 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         # deprecated, will be removed in 1.6
         self._loaded_optimizer_states_dict = {}
 
+        self._prevent_trainer_and_dataloaders_deepcopy: bool = False
+
```

Comment:
Can you move this to L112 with the rest of non-deprecated attributes?

Reply:
Done !